frontend/src/components/Alerts/FailedAlertContent.tsx

```diff
@@ -31,7 +30,7 @@ export const FailedAlertContent = ({ title, message }: { title: string; message:
                 <Typography>{TranslateText(title)}</Typography>
             </StyledAlertTitle>
             <Indent>
-                <TextAlignedButton variant="ghost" color="secondary" style={{ backgroundColor: bannerColor }}>
+                <TextAlignedButton variant="ghost" color="secondary" style={{ opacity: '1' }}>
```

Comment:
```suggestion
                <TextAlignedButton variant="ghost" color="secondary">
```

Doesn't seem to have impact. Looks like 1 is default